keras/backend.py

```diff
@@ -5647,7 +5651,7 @@ def categorical_focal_crossentropy(
     )
 
     if from_logits:
-        output = tf.nn.softmax(output, axis=axis)
+        output = softmax(output, axis=axis)
```

Comment:
I think redirect to an extra level of internal function is unnecessary. Please keep the existing code.